### ISSUE TITLE
Cannot connect to ssh server with banner

### ISSUE THREAD

**Issue by elehcim** [wavetermdev/waveterm#179]
**Describe the bug**
Error when connecting to a server with banner set:
in the server ssh config this options is set:
```
Banner /etc/mybanner
```
where `/etc/mybanner` is something like:
```
==========================================================
welcome
==========================================================
```

In Wave I get:
```
prompt> error connecting to remote: invalid packet received from mshell client: raw[+==========================================================+]
prompt> error reading from controlling-pty: read /dev/ptmx: input/output error
```

**To Reproduce**
Add a connection

**Expected behavior**
Connection successful

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Linux x64
 - Version  v0.5.2

**Comment by sawka** [comment]
thanks for posting!  i'll look into this.  i put a fix in for something i thought was similar to this issue in v0.5.2, but i see you're still running into it.  this looks easy to reproduce though (just by setting the ssh banner), so i'll check it out.

**Comment by oneirocosm** [comment]
This has been fixed in v0.7.0 with the caveat that the banner isn't shown. So we may want to open a separate issue for that later.
